test/pylib/scylla_cluster.py

```diff
@@ -220,6 +221,9 @@ def __init__(self, mode: str, exe: str, vardir: str,
                  property_file: Dict[str, Any],
                  append_env: Dict[str,Any]) -> None:
         # pylint: disable=too-many-arguments
+        # this variable needed to make a cleanup after server is not needed anymore
+        self.maintenance_socket_dir = tempfile.TemporaryDirectory()
```

Comment:
Please add `prefix=scylladb-test.py-` so if a directory like that is orphaned (say python terminates unexpectedly) users will know what that is and that it's safe to remove it.

Reply:
Done. For better clarity, I added the server ID as well.